util/configure

```diff
@@ -1071,7 +1066,8 @@ _EOC_
         }
     }
 
-    # configure opm:
+    # configure opm
```

Comment:
Why this change?

Reply:
Consistency with other comments.  It was sort of an accidental inclusion.  Same for below.  Most comments don't have colons - though actually they're not totall consisten.